bai-tap-buoi-9/main.js

```diff
@@ -1 +1,151 @@
-const 
\ No newline at end of file
+// Hàm kiểm tra số nguyên
+function checkNumber(number) {
+    if (parseInt(number)==number) {
+        return true;
+    }
+    else {
+        return false;
+    }
+}
+// Bài 1:
+const realSalary = (money) => {
```

Comment:
```suggestion
const getSalary = salary => {
```
Tên hàm phải là động từ
Hàm có 1 param thì có thể bỏ dấu `()`